lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -55,7 +28,7 @@ def to_representation(self, course_overview):
             start_type = "empty"
             start_display = None
```

Comment:
Yes, this is the refactor code that I mentioned on your PR that is still pending.